CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+- Collect information about the bug:
+  - Stack trace (Traceback) or screenshot error message
+  - OS, Platform and Version (Windows, Linux, macOS, x86, ARM)
+  - Input dataset, can you recreate this bug with one of the example datasets provided by `glidertest`? If not, provide your dataset if practical
+- Open an [Issue](https://github.com/callumrollo/glidertest/issues) describing the bug
+
+<!-- omit in toc -->
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for XYZ, **including completely new features and minor improvements to existing functionality**. 
+
+<!-- omit in toc -->
+#### Before Submitting an Enhancement
+
+- Make sure that you are using the latest version.
+- Read the [documentation](https://callumrollo.github.io/glidertest) carefully and find out if the functionality is already covered, maybe by an individual configuration.
+- Find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince the project's developers of the merits of this feature. Keep in mind that we want features that will be useful to the majority of our users and not just a small subset.
+
+<!-- omit in toc -->
+#### How Do I Submit a Good Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://github.com/callumrollo/glidertest/issues).
+
+- Use a **clear and descriptive title** for the issue to identify the suggestion.
+- Provide a **step-by-step description of the suggested enhancement** in as many details as possible.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why. At this point you can also tell which alternatives do not work for you.
+- **Explain why this enhancement would be useful** to most XYZ users. 
+
+### Your First Code Contribution
+
+Getting started adding your own functionality.
+
+#### glidertest organisation
+
+Code is organised into files within `glidertest/*.py` and demonstrated in jupyter notebooks in `notebooks/*.ipynb`. The *.py* files include primarily functions (with their necessary packages imported) while the notebooks call these functions and display the plots generated. The *.py* files are separated into broad categories of plots, tools and utilities. If you'd like to add a function to calculate some thing WIDGET and then to plot the result of the calculation, you will want to write a function in tools.py and then a plotting function in plots.py. There are a couple exceptions: If it's a very simple calculation (mean, median, difference between two quantities), you might put the entire calculation within the plotting function. See for example `plots.plot_updown_bias()`. Or, if the calculation is more complicated, but is easily displayed by an existing function, then you might have a calculation function `tools.calc_foo_bar()` and then use an existing `plots.plot_histogram()` to display it.
```

Comment:
`plots.plot_updown_bias()` does not have any calculations within. `process_optics_assess `does though

Reply:
good catch!